features/duplicatecheck.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Duplicate users/orgs checking
+	As a developer
+	So that there is no duplicate uers/orgs
+	I want the app to check duplicate users/org
+  
+Background: User is trying to sign up
+    Given the following orgs exist:
+        | name | contact_id |
+        | org1 | 1          |
+        | org2 | 2          |
+        | org3 | 3          |
+
+    And the following users exist:
+        | name  | github_uid      | email          |
+        | user1 | esaas_developer | user1@user.com |
+        | user2 |                 | user2@user.com |
+        | user3 |                 | user3@user.com |
+    
+    And I'm logged in on the orgs page
+
+Scenario: user fills in information in new user page
+    Given I am on the Users page
+    When I follow "New User"
+    And I fill in "User Name" with "randomuser"
+    And I fill in "User E-mail address" with "user1@user.com"
+    And I press "Save"
+	Then I should see "User E-mail address has already been taken"
+	And I should not see "User successfully created."
+	
+Scenario: user fills in information in new user page
+    Given I am on the Users page
+    When I follow "New User"
+    And I fill in "User Name" with "new_user"
+    And I fill in "User E-mail address" with "new_user@user.com"
+    And I press "Save"
+	Then I should see "User successfully created."
```

Comment:
SMART test! As a small edit I would recommend you change the indentation so it all looks similar and make it even more beautiful.